lib/elixir/lib/module/parallel_checker.ex

```diff
@@ -158,8 +159,50 @@ defmodule Module.ParallelChecker do
     :gen_server.call(server, {:unlock, module})
   end
 
+  defp prepare_modules(modules) do
+    Enum.map(modules, fn {map, binary} ->
+      {:ok, _module, chunks} = :beam_lib.all_chunks(binary)
+      map = fill_missing_map_fields(map, chunks)
```

Comment:
So here is the part I am missing: isn't this going to be called for all modules? And therefore won't we load the chunk for all modules?

Reply:
We only fill in missing map fields and only runtime modules will have missing fields.

Reply:
Do, I missed the pattern matching. Thanks!